sklearn/datasets/twenty_newsgroups.py

```diff
@@ -216,11 +216,18 @@ def fetch_20newsgroups(data_home=None, subset='train', categories=None,
 
     Returns
     -------
-    bunch : Bunch object
-        bunch.data: list, length [n_samples]
-        bunch.target: array, shape [n_samples]
-        bunch.filenames: list, length [n_classes]
-        bunch.DESCR: a description of the dataset.
+    bunch : Bunch object with the following attribute:
+
+        - bunch.data: list, length [n_samples]
+
+        - bunch.target: array, shape [n_samples]
+
+        - bunch.filenames: list, length [n_classes]
+
+        - bunch.DESCR: a description of the dataset.
+
+        - bunch.target_names: a list of categories of the returned data,
+          length [n_classes]. This depends of the `categories` parameter.
```

Comment:
Is this another one in the itemized list, or is it a second output (in which case it has one extra indentation)?

Reply:
in the last commit I improve this